src/widgets/HelpSystem.cpp

```diff
@@ -289,15 +289,15 @@ void HelpSystem::ShowHelp(wxWindow *parent,
       // I can't find it'.
       // Use Built-in browser to suggest you use the remote url.
       wxString Text = HelpText( wxT("remotehelp") );
-      Text.Replace( wxT("*URL*"), remoteURL );
+      Text.Replace( wxT("*URL*"), remoteURL.GET() );
       // Always make the 'help on the internet' dialog modal.
       // Fixes Bug 1411.
       ShowHtmlText( parent, XO("Help on the Internet"), Text, false, true );
    }
    else if( HelpMode == wxT("Local") || alwaysDefaultBrowser)
    {
       // Local file, External browser
-      OpenInDefaultBrowser( wxString(wxT("file:"))+localFileName );
+      OpenInDefaultBrowser( L"file:" + localFileName );
```

Comment:
In this case has a possibility get a false by return and should we processing this case? For example, like:
```cpp
MessgeBox(
  "Caption",
  wxString("Can't open link in browser, please follow to this link for help reading:") + uri.BuildURI());
```
and remake `void OpenInDefaultBrowser()` to `bool OpenInDefaultBrowser()`;

But may be this is extra.

Reply:
You may be right, but my commit is neutral.  It leaves existing behavior the same.